portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/CookiesUseCase.testcase

```diff
@@ -2,6 +2,7 @@
 definition {
 
 	property ci.retries.disabled = "true";
+	property custom.properties = "feature.flag.LPS-142518=true${line.separator}jsonws.web.service.paths.excludes=";
```

Comment:
@melodywu  
If the changes contains `feature.flag`, SF will enforce checking `portal-impl/src/portal.properties`.
I send the fix here: https://github.com/ling-alan-huang/liferay-portal/pull/1669

Reply:
thank you Alan.